### ISSUE TITLE
Add a contact name takes two clicks, when it should just be one

### ISSUE THREAD

**Issue by stringhandler** [tari-project/wallet-ios#1116]
On the transaction details screen, when you clickAdd contact, it should go straight to typing in the name. Instead you must click Edit before editing the name.

Screen 1
![image](https://github.com/user-attachments/assets/8c9956f5-ab43-48f1-a145-ccc489c718ee)

Screen 2
![image](https://github.com/user-attachments/assets/c20cdb07-cce4-4d5f-8c01-2e153c6637de)

Screen 3
![image](https://github.com/user-attachments/assets/21c01ec5-1d9b-4aff-9715-aca35f5acaf7)

In other words, the current flow is Screen 1 -> 2 -> 3. It should go from 1 -> 3

**Comment by TruszczynskiA** [comment]
This change will be considered when user interface of this screen will be updated.
